tests/test_experiment_rsmpredict.py

```diff
@@ -52,8 +52,8 @@ def test_run_experiment_parameterized(*args, **kwargs):
     check_run_prediction(*args, **kwargs)
 
 
+# Check that both rsmtool and rsmpredict generate the same files
```

Comment:
Just to be consistent with the other tests where the name of the tests are shown when running the tests rather than the docstring. Also, given that this is a parameterized tests, it would be good to see the parameters.

Reply:
I see! I was confused because all the other test functions in the file had docstrings. Thank you!